brave-spancollector-kafka/src/test/java/com/github/kristofa/brave/kafka/ITKafkaSpanCollector.java

```diff
@@ -0,0 +1,83 @@
+package com.github.kristofa.brave.kafka;
+
+import com.github.kristofa.brave.SpanCollector;
+import com.twitter.zipkin.gen.Span;
+import kafka.consumer.ConsumerIterator;
+import kafka.consumer.ConsumerTimeoutException;
+import kafka.consumer.KafkaStream;
+import kafka.javaapi.consumer.ConsumerConnector;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.thrift.TDeserializer;
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.*;
+import static org.junit.Assert.assertEquals;
+
+public class ITKafkaSpanCollector {
+
+    private final static String TOPIC = "zipkin";
```

Comment:
I'd move the TOPIC constant name zipkin to somewhere in the main tree, package private, then use it everywhere.